emscripten.py

```diff
@@ -15,6 +15,7 @@
 from tools import jsrun, cache as cache_module, tempfiles
 from tools.response_file import read_response_file
 from tools.shared import WINDOWS
+from tools.toolchain_profiler import *
```

Comment:
It is used, but e.g. that file already accessed `Popen` without `subprocess.` prefix, here: https://github.com/juj/emscripten/blob/5b0abedb3ed83a294832575117a0ba06de6d9d21/emscripten.py#L142, so the import monkeypatches `Popen` to the profiled version.

Also in the second part of the PR, there are some profiling blocks that I added, so it gets more use, like https://github.com/juj/emscripten/blob/toolchain_profiler/emscripten.py#L156